src/ensembl/utils/plugin.py

```diff
@@ -194,6 +197,9 @@ def test_method(..., test_dbs: dict[str, UnitTestDB], ...):
         if src is not None:
             src = Path(src)
         name = argument.get("name", None)
-        key = name if name else src.name
+        try:
+            key = name if name else src.name
```

Comment:
curious if we can instead use or operator here 
```suggestion
            key = name or src.name
```
```suggestion
            key = name | src.name
```

Reply:
This is a new trick I didn't know: it seems `|` does not work if either is `None`, but `or` does!